aws/requests.yaml

```diff
@@ -1,20 +1,26 @@
 releases:
     - name: ">= 15.0.0"
       requests:
+        - name: app-operator
+          version: ">= 4.1.0"
+          issue: https://github.com/giantswarm/roadmap/issues/254
         - name: cert-operator
           version: ">= 1.0.1"
           issue: https://github.com/giantswarm/giantswarm/issues/14205
         - name: cluster-operator
-          version: ">= 3.6.1"
+          version: ">= 3.7.1"
           issue: https://github.com/giantswarm/giantswarm/issues/14205
         - name: coredns
           version: ">= 1.4.0"
           issue: https://github.com/giantswarm/giantswarm/issues/13771
     - name: "> 14.0.0"
       requests:
         - name: app-operator
-          version: ">= 3.2.0"
+          version: ">= 3.2.1 < 4.0.0"
           issue: https://github.com/giantswarm/giantswarm/issues/15392
+        - name: chart-operator
+          version: ">= 2.10.0"
```

Comment:
This includes a fix for helm creating too many release secrets when a chart fails to install.